src/Control/Controller.php

```diff
@@ -332,14 +326,10 @@ public function getURLParams()
     /**
      * Returns the HTTPResponse object that this controller is building up. Can be used to set the
      * status code and headers.
-     *
-     * @return HTTPResponse
      */
-    public function getResponse()
+    public function getResponse(): HTTPResponse
     {
-        if (!$this->response) {
-            $this->setResponse(new HTTPResponse());
-        }
+        $this->response ??= HTTPResponse::create();
```

Comment:
I actually prefer how this was before - but ultimately it's an aesthetic choice so I won't ask you to revert it.

Reply:
Haven't seen this use of null coalescing operator before.

Reply:
This has been changed to `$this->setResponse(HTTPResponse::create());` in the constructor